src/components/Home.js

```diff
@@ -0,0 +1,37 @@
+import React from "react";
+
+export default function Home() {
+    return (
+        <div>
```

Comment:
you can use a Fragment here .. no need for the div unless you need it for the DOM structure and styling